aspnetcore/release-notes/aspnetcore-10/includes/openApi.md

```diff
@@ -34,6 +37,10 @@ OpenAPI 3.1 support was primarily added in the following [PR](https://github.com
 ### OpenAPI 3.1 breaking changes
 
 Support for OpenAPI 3.1 requires an update to the underlying OpenAPI.NET library to a new major version, 2.0. This new version has some breaking changes from the previous version. The breaking changes may impact apps if they have any document, operation, or schema transformers.
+Breaking changes in this iteration include the following:
+
+* Entities within the OpenAPI document, like operations and parameters, are typed as interfaces. Concrete implementations exist for the inlined and referenced variants of an entity. For example, an `IOpenApiSchema` can either be an inlined `OpenApiSchema` or an `OpenApiSchemaReference` that points to a schema defined elsewhere in the document.
```

Comment:
I left a comment similar to this here: https://github.com/microsoft/OpenAPI.NET/pull/2298#issuecomment-2771826088

Like @Rick-Anderson said, this item can probably be copy-pasted right into their docs too.

Reply:
> 
> Like @Rick-Anderson said, this item can probably be copy-pasted right into their docs too.

@martincostello  can you recommend which doc and section?

Reply:
I'm not sure about the section, but this is the document: https://github.com/microsoft/OpenAPI.NET/blob/main/docs/upgrade-guide-2.md